ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx

```diff
@@ -112,8 +108,23 @@ export function getComplianceLabelGroupColor(
     return 'blue';
 }
 
+export function sortCheckStats(items: ComplianceCheckStatusCount[]): ComplianceCheckStatusCount[] {
+    const order: ComplianceCheckStatus[] = [
+        'PASS',
+        'FAIL',
+        'MANUAL',
+        'ERROR',
+        'INFO',
+        'NOT_APPLICABLE',
+        'INCONSISTENT',
+    ];
```

Comment:
Since even paraniods have enemies, they might add `UNSET_CHECK_STATUS` as last item in `order` array.